models/image_classification_mobilenet/mobilenet.cpp

```diff
@@ -0,0 +1,140 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/dnn.hpp>
+#include <opencv2/core/utils/logger.hpp>
```

Comment:
It is not used right? I suggest using one header to replace all of these to keep it simple:

```
#include "opencv2/opencv.hpp"
```